packages/ui/src/components/Pagination/PaginationButtons.tsx

```diff
@@ -0,0 +1,153 @@
+import styled from '@emotion/styled'
+import { useCallback, useMemo } from 'react'
+import { Button } from '../Button'
+import { Text } from '../Text'
+import { getPageNumbers } from './getPageNumbers'
+
+const PageNumbersContainer = styled.div`
```

Comment:
Could we not use Row and Stack around these? Maybe styled but at least it would reduce CSS